src/main/java/org/craftercms/studio/api/v1/service/deployment/DeploymentService.java

```diff
@@ -73,36 +37,26 @@ List<ContentItemTO> getScheduledItems(String site, String sort, boolean ascendin
      *
      * @throws ServiceLayerException exception is case of en error
      */
-    void bulkGoLive(String site, String environment, String path, String comment) throws ServiceLayerException;
+    long bulkGoLive(String site, String environment, String path, String comment) throws ServiceLayerException, AuthenticationException;
```

Comment:
Let's indicate what the `long` is for in the JavaDoc.

Reply:
Added to the javadoc. The returned long is the id of the create publish package